src/portable_python/cpython.py

```diff
@@ -228,7 +228,13 @@ def _finalize(self):
 
                 self.run_python(cmd)
 
-            self.run_python("-mpip", "install", *runez.flattened(additional))
+            self.run_python(
+                "-mpip",
+                "install",
+                "--no-cache-dir",
+                "--upgrade",
```

Comment:
You probably want `--upgrade` because you'd like to ensure setuptools is latest?
If so, note that the modules in `<python-installation>/lib/python3.11/site-packages/` are a bit "special", they work in a subtle way... they don't matter when you create a venv for example... They only apply to runs using directly `<python-installation>/bin/python3`

Reply:
Thanks, yeah, I know venv can be created with a full different set of pip, setuptools and wheel. I just want to update the ones for the original python3 binary.

Reply:
I see the formatter is different with black. It is okay to follow this project's preference.

Reply:
> I see the formatter is different with black. It is okay to follow this project's preference.

Just update for linter.